redash/cli/rq.py

```diff
@@ -22,6 +23,11 @@ def scheduler():
 @manager.command()
 @argument('queues', nargs=-1)
 def worker(queues='default'):
+    # Configure any SQLAlchemy mappers loaded until now so that the mapping configuration 
+    # will already be available to the forked work horses and they won't need 
+    # to spend valuable time re-doing that on every fork.
+    configure_mappers()
```

Comment:
I've noticed that in `redash.models.base` we use the `flask_sqlalchemy.SQLAlchemy`'s instance `configure_mappers`. I wonder what's the difference?

Reply:
[It's the same](https://github.com/pallets/flask-sqlalchemy/blob/master/flask_sqlalchemy/__init__.py#L73).

Reply:
WTF. Why did they do that? :confused:

Reply:
So you can access everything through `flask_sqlalchemy.SQLAlchemy`, I guess.

Reply:
Yeah, the idea being that you define a db object like `db = SQLAlchemy()` when using flask-sqlalchemy and then can call anything inside the `sqlalchemy.orm` module. In other words, it's to simplify the Flask-like API of flask-sqlalchemy.